src/main/java/de/zalando/aruha/nakadi/repository/TopicRepository.java

```diff
@@ -2,30 +2,27 @@
 
 import de.zalando.aruha.nakadi.domain.BatchItem;
 import de.zalando.aruha.nakadi.domain.Cursor;
+import de.zalando.aruha.nakadi.domain.EventTypeStatistics;
 import de.zalando.aruha.nakadi.domain.Topic;
 import de.zalando.aruha.nakadi.domain.TopicPartition;
 import de.zalando.aruha.nakadi.exceptions.DuplicatedEventTypeNameException;
 import de.zalando.aruha.nakadi.exceptions.EventPublishingException;
 import de.zalando.aruha.nakadi.exceptions.NakadiException;
 import de.zalando.aruha.nakadi.exceptions.TopicCreationException;
 import de.zalando.aruha.nakadi.exceptions.TopicDeletionException;
-
 import java.util.List;
 import java.util.Map;
 
 /**
  * Manages access to topic information.
  *
- * @author  john
+ * @author john
  */
 public interface TopicRepository {
 
     List<Topic> listTopics() throws NakadiException;
 
-    void createTopic(String topic) throws TopicCreationException, DuplicatedEventTypeNameException;
-
-    void createTopic(String topic, int partitionsNum, int replicaFactor, long retentionMs, long rotationMs)
-            throws TopicCreationException, DuplicatedEventTypeNameException;
+    void createTopic(String topic, EventTypeStatistics defaultStatistics) throws TopicCreationException, DuplicatedEventTypeNameException;
```

Comment:
I would be ok with changing the API to receive the whole `EventType` object instead of adding a separate parameter. Wdyt?